src/api/app/controllers/webui/project_controller.rb

```diff
@@ -85,6 +85,8 @@ def users
   def subprojects
     @subprojects = @project.subprojects.order(:name)
     @parentprojects = @project.ancestors.order(:name)
+    @parent = @project.parent
```

Comment:
Hmm, you only need this for getting the name length for the view right? A ``pluck`` would be nice but as this is a function it is probably not that easy ... :disappointed: